docs/build/build-ss58-registry.md

```diff
@@ -6,7 +6,7 @@ description: Information about SS58 encoding.
 slug: ../build-ss58-registry
 ---
 
-:::note Substrate-based chains use an [SS58 encoding](https://docs.substrate.io/v3/advanced/ss58/) for their address formats.
+:::note Substrate-based chains use an SS58 encoding for their address formats.
```

Comment:
Are there plans to add a dedicated page on SS58 encoding? If not, we can link it to https://docs.substrate.io/main-docs/fundamentals/accounts-addresses-keys/#address-encoding-and-chain-specific-addresses

Reply:
Just saw the issue you linked